content/300-guides/200-deployment/301-edge/450-deploy-to-cloudflare-workers.mdx

```diff
@@ -1,75 +1,44 @@
+- [Prisma Data Platform](https://console.prisma.io/) account
+- Node.js & npm installed
 - Git installed
 
-## 1. Get a MongoDB Atlas connection string
-
-After you create a MongoDB Atlas database cluster, you can get its connection string. By default, that connection string does not include a database name.
-
-When you import your project in the Prisma Data Platform at [step 6](#6-import-your-project-into-the-prisma-data-platform), you must provide a connection string that includes a database name.
-
-However, you do not need to manually create a database in MongoDB Atlas. The project creation workflow in the Prisma Data Platform will create a database with the name you define in the connection string.
-
-1. [Create a free shared cluster](https://www.mongodb.com/docs/atlas/tutorial/create-new-cluster/) in MongoDB Atlas.
-2. On the _Security Quickstart_ screen, complete the initial security configuration.
-   1. Add a new database user.
-   2. Click **Add My Current IP Address** so that you can connect to the MongoDB shared cluster from your current system.
-   3. Click **Finish and Close**.
-3. [Get the database cluster connection string](https://www.mongodb.com/docs/guides/atlas/connection-string/).
-4. Paste the connection string in a text editor and add the collection name `Log`.
-
-   The code sample below shows a connection string, that is originally provided by MongoDB Atlas, with the database name now added.
-
-   ```diff
-   # The first line is a default-style MongoDB Atlas connection string.
-   # The second line includes the inserted `Log` database name.
-   - mongodb+srv://<user>:<password>@cluster.mongodb.net/?retryWrites=true&w=majority
-   + mongodb+srv://<user>:<password>@cluster.mongodb.net/Log?retryWrites=true&w=majority
-   ```
-
-You now have a complete connection string. You will paste the connection string when you create your project in [6. Import your project into the Prisma Data Platform](#6-import-your-project-into-the-prisma-data-platform).
-
-For security reasons, do not keep the connection string in the text editor or save it in a text file.
-
-## 2. Set up your application
-
 Open your terminal and navigate to a location of your choice. Run the following commands to set up your application.
 
 ```terminal
-mkdir prisma-mongodb-cloudflare
-cd prisma-mongodb-cloudflare
+mkdir prisma-cloudflare-accelerate
+cd prisma-cloudflare-accelerate
 npm init -y
 npm install -D prisma typescript wrangler
 ```
 
-## 3. Set up Wrangler
+## 1. Set up Wrangler
 
-Wrangler is the official Cloudflare Worker CLI. You will use it to develop and deploy to Cloudflare Workers. This guide uses [Wrangler v2](https://developers.cloudflare.com/workers/wrangler/).
+Wrangler is the official Cloudflare Worker CLI. You will use it to develop and deploy to Cloudflare Workers. This guide uses [Wrangler v3](https://developers.cloudflare.com/workers/wrangler/).
 
 First, initialize Wrangler. To do this, run the following command in your terminal:
```

Comment:
The output when a user runs `npx wrangler init` need an update. The prompts are different in wrangler version 3.

Reply:
```
npx wrangler init
 ⛅️ wrangler 3.14.0
-------------------
Using npm as package manager, as there is already a package-lock.json file.
▲ [WARNING] The `init` command is no longer supported. Please use `npm create cloudflare@2` instead.

  The `init` command will be removed in a future version.


Running `npm create cloudflare@2`...

using create-cloudflare version 2.0.8

╭ Create an application with Cloudflare Step 1 of 3
│ 
├ Where do you want to create your application?
│ dir plain-dew-29c8
│
├ What type of application do you want to create?
│ type Website or web app
│
├ Which development framework do you want to use?
│ framework Hono
│
╰ Continue with Hono via `npx create-hono@0.2.0 plain-dew-29c8 --template cloudflare-pages`

Need to install the following packages:
create-hono@0.2.0
Ok to proceed? (y) 

create-hono version 0.1.1
cloned honojs/starter#main to /Users/ruheni/Documents/repos/prisma/temp/prisma-cloudflare-accelerate/plain-dew-29c8
✔ Copied project files

╭ Configuring your application for Cloudflare Step 2 of 3
│ 
├ Installing wrangler A command line tool for building Cloudflare Workers 
│ installed via `npm install wrangler --save-dev`
│ 
├ Adding command scripts for development and deployment
│ added commands to `package.json`
│ 
╰ Application configured 

╭ Deploy with Cloudflare Step 3 of 3
│ 
├ Do you want to deploy your application?
│ no deploying via `npm run deploy`
│
├  APPLICATION CREATED  Deploy your application with npm run deploy
│ 
│ Run the development server npm run dev
│ Deploy your application npm run deploy
│ Read the documentation https://developers.cloudflare.com/pages
│ Stuck? Join us at https://discord.gg/cloudflaredev
│ 
╰ See you again soon! 
```